go-controller/pkg/ovn/loadbalancer.go

```diff
@@ -43,32 +43,8 @@ func (ovn *Controller) getLoadBalancer(protocol kapi.Protocol) (string, error) {
 	return out, nil
 }
 
-// getDefaultGatewayLoadBalancer returns the load balancer for the node with the lowest gateway IP.
-// This is used in the implementation of ExternalIPs
-func (ovn *Controller) getDefaultGatewayLoadBalancer(protocol kapi.Protocol) string {
-	if outStr, ok := ovn.loadbalancerGWCache[protocol]; ok {
-		return outStr
-	}
-
-	gw, _, err := util.GetDefaultGatewayRouterIP()
-	if err != nil {
-		klog.Errorf(err.Error())
-		return ""
-	}
-
-	externalIDKey := string(protocol) + "_lb_gateway_router"
-	lb, _, _ := util.RunOVNNbctl("--data=bare",
-		"--no-heading", "--columns=_uuid", "find", "load_balancer",
-		"external_ids:"+externalIDKey+"="+gw)
-	if len(lb) != 0 {
-		ovn.loadbalancerGWCache[protocol] = lb
-		ovn.defGatewayRouter = gw
-	}
-	return lb
-}
-
-// getLoadBalancerVIPs returns a map whose keys are VIPs (IP:port) on loadBalancer
-func (ovn *Controller) getLoadBalancerVIPs(loadBalancer string) (map[string]interface{}, error) {
+func (ovn *Controller) getLoadBalancerVIPs(
+	loadBalancer string) (map[string]interface{}, error) {
```

Comment:
this change is not needed?

Reply:
You mean the new line? Yes, it's not _needed_...but should we really have method definitions which jump several lines?